roads.mss

```diff
@@ -3100,3 +3100,81 @@ tertiary is rendered from z10 and is not included in osm_planet_roads. */
     }
   }
 }
+#railways-text-name {
+  /* Others started from z15. */
```

Comment:
In Manhattan NYC, z14 ~ z16 are really too cluttered, no much rooms to render subway labels :cry:.

How about:
- High-speed from z11
- Main rail lines from z14
- Others from z17?